File: java/src/Rosliny/Trawa.java
package Rosliny;
import Swiat.*;

public class Trawa extends Roslina{
    public Trawa(int x, int y, Swiat swiat){super(Gatunek.TRAWA, 0, 0, x, y, swiat);}
    public Trawa(int x, int y, int sila, Swiat swiat){super(Gatunek.TRAWA, sila, 0, x, y, swiat);}

    @Override
    public String rysowanie(){return "T";}
}

File: python/plants/trawa.py
from plants.roslina import Roslina


class Trawa(Roslina):

    def __str__(self):
        return self._species + "[" + str(self._x) + "][" + str(self._y) + "]"

    def __init__(self, x, y, world, sila=0):
        Roslina.__init__(self, "trawa", sila, 0, x, y, world)

    def rysowanie(self):
        return "T"
